soccer/motion/MotionControl.cpp

```diff
@@ -240,3 +240,17 @@ void MotionControl::_targetBodyVel(Point targetVel) {
     _robot->control->set_xvelocity(targetVel.x());
     _robot->control->set_yvelocity(targetVel.y());
 }
+
+Pid* MotionControl::getPid(char controller) {
+    // just in case there is confusion
+    switch (tolower(controller)) {
+        case 'a':
```

Comment:
It would probably be easier to fix after comp especially since I also have no clue how enums would work across the bridge.